frontend/src/metabase/common/components/UserCollectionList.tsx

```diff
@@ -28,63 +21,36 @@ export const UserCollectionList = () => {
   const users = data?.data ?? [];
   const total = data?.total;
 
-  return (
-    <Flex direction="column" p="1.5rem" h="100%">
-      <ListHeader>
-        <BrowserCrumbs
-          crumbs={[
-            {
-              title: ROOT_COLLECTION.name,
-              to: Urls.collection({ id: "root", name: "" }),
-            },
-            { title: PERSONAL_COLLECTIONS.name },
-          ]}
-        />
-      </ListHeader>
-      <Box style={{ flexGrow: 1, overflowY: "auto" }} pr="0.5rem">
-        {isLoading ? (
-          <Flex justify="center" align="center" h="100%">
-            <Loader size="lg" />
-          </Flex>
-        ) : (
-          <Grid>
-            {users.map(
-              (user) =>
-                user.personal_collection_id && (
-                  <ListGridItem
-                    key={user.personal_collection_id}
-                    role="list-item"
-                  >
-                    <Link to={`/collection/${user.personal_collection_id}`}>
-                      <Card shadow="none" withBorder>
-                        <Group gap="xs">
-                          <Icon
-                            name="person"
-                            className={CS.mr1}
-                            c="text-medium"
-                            size={18}
-                          />
+  const crumbs = [
+    {
+      title: ROOT_COLLECTION.name,
+      to: Urls.collection({ id: "root", name: "" }),
+    },
+    { title: PERSONAL_COLLECTIONS.name },
+  ];
 
-                          <h3>{user.common_name}</h3>
-                        </Group>
-                      </Card>
-                    </Link>
-                  </ListGridItem>
-                ),
-            )}
-          </Grid>
-        )}
-      </Box>
-      <Flex justify="end" className={CS.syncStatusAwarePagination}>
-        <PaginationControls
-          page={page}
-          pageSize={PAGE_SIZE}
-          total={total}
-          itemsLength={PAGE_SIZE}
-          onNextPage={handleNextPage}
-          onPreviousPage={handlePreviousPage}
-        />
-      </Flex>
-    </Flex>
+  const items = users
+    .filter((user) => user.personal_collection_id)
+    .map((user) => ({
+      key: user.personal_collection_id!,
+      name: user.common_name,
+      icon: "person" as const,
```

Comment:
Should we use the correct type? I mean it will help you with autocompletion too if you ever need to change this value again.
```suggestion
      icon: "person" as IconName,
```

Reply:
`as const` keeps type safety while `as IconName` is an unsafe cast which means it might hide errors.
See no errors here:
<img width="462" height="296" alt="image" src="https://github.com/user-attachments/assets/aabb4ea3-6322-4033-8303-823a427cce00" />
But it errors here:
<img width="466" height="304" alt="image" src="https://github.com/user-attachments/assets/5855d67e-a0ea-48bf-90a1-399cc20a13db" />

Reply:
Ok we probably want `satisfies IconName` instead. I didn't want to do type cast, but use a property that will also make autocompletion works.